api/clients/buckets/bucket.go

```diff
@@ -100,469 +39,63 @@ func WithRetrySettings(maxRetries int, durationBetweenTries time.Duration, maxWa
 //
 // Returns:
 //   - *Client: A pointer to a new Client instance initialized with the provided rest.Client and logger.
-func NewClient(client *rest.Client, option ...Option) *Client {
+func NewClient(client *rest.Client) *Client {
 	client.SetHeader("Cache-Control", "no-cache")
 
 	c := &Client{
 		client: client,
-		retrySettings: retrySettings{
-			maxRetries:           15,
-			durationBetweenTries: time.Second,
-			maxWaitDuration:      2 * time.Minute,
-		},
-	}
-
-	for _, o := range option {
-		o(c)
 	}
 
 	return c
 }
 
-// Get retrieves a bucket definition based on the provided bucketName. The function sends a GET request
```

Comment:
Note: I've omitted documentation for now, this initial change is just so we can review the concept.